moveit_core/constraint_samplers/src/default_constraint_samplers.cpp

```diff
@@ -565,6 +555,26 @@ bool constraint_samplers::IKConstraintSampler::sampleHelper(robot_state::RobotSt
       return false;
     }
 
+    // we now have the transform we wish to perform IK for, in the planning frame
+    if (transform_ik_)
+    {
+      // we need to convert this transform to the frame expected by the IK solver
+      // both the planning frame and the frame for the IK are assumed to be robot links
```

Comment:
How would you rephrase it?
The comment was there before the patch,
so I guess @hartmanndennis didn't give it too much thought whether it should be rephrased. :-)